public/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="author" content="Katie Tracy" />
+  <meta name="description" content="Find a donation fund worldwide to support coronavirus relief efforts." />
+  <meta name="keywords"
+    content="donate, donation, covid19, coronavirus, novel coronavirus, pandemic, outbreak, relief, emergency, aid" />
+  <meta name="theme-color" content="#f2994a" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <meta name="og:image" content="https://donatecovid19.org/img/embed.png" />
+  <!-- ^^ kind of a weird fix but it works for now -->
+  <meta name="og:title" content="Donate to Fight COVID-19" />
+
+  <meta name="twitter:url" content="https://donatecovid19.org">
+  <meta name="twitter:title" content="Donate to Fight COVID-19">
+  <meta name="twitter:description" content="Find a donation fund worldwide to support coronavirus relief efforts.">
```

Comment:
Nit: twitter?

Reply:
Just to pretty up the Twitter embed - Twitter does embeds diff. than most platforms